app/resources/locale/en/en.po

```diff
@@ -1246,90 +1235,108 @@ msgstr "Wrong block format: {{type}}"
 msgid "Wrong block name {{name}}"
 msgstr "Wrong block name {{name}}"
 
-#: app/scripts/controllers/menu.js:1907
-#, fuzzy
+#: app/scripts/controllers/menu.js:1925
 msgid "Wrong new name!"
-msgstr "Wrong port name {{name}}"
+msgstr "Wrong new name!"
 
-#: app/scripts/services/project.js:143
+#: app/scripts/services/project.js:154
 msgid "Wrong project format: {{name}}"
 msgstr "Wrong project format: {{name}}"
 
-#: app/scripts/services/tools.js:862
+#: app/scripts/services/tools.js:973
 msgid "Wrong remote hostname {{name}}"
 msgstr "Wrong remote hostname {{name}}"
 
-#: app/scripts/controllers/menu.js:1811
+#: app/scripts/controllers/menu.js:1829
 msgid "Wrong search name!"
-msgstr ""
+msgstr "Wrong search name!"
 
-#: app/scripts/services/forms.js:412 app/views/design.html:166
+#: app/scripts/services/forms.js:412 app/views/design.html:164
 msgid "Yellow"
 msgstr "Yellow"
 
-#: app/scripts/controllers/menu.js:394
-#, fuzzy
+#: app/scripts/controllers/menu.js:409
+msgid ""
+"You are editing a submodule, if you save it, you'll save only               "
+"the submodule (in this situation \"save as\" works like               "
+"\"export module\"). Do you want to continue?"
+msgstr ""
+"You are editing a submodule, if you save it, you'll save only               "
+"the submodule (in this situation \"save as\" works like               "
+"\"export module\"). Do you want to continue?"
+
+#: app/scripts/controllers/menu.js:396
 msgid ""
-"You are editing a submodule, if you save it, you save only               the "
-"submodule (in this situation \"save as\" works like               \"export "
-"module\"), Do you like to continue?"
+"You are navigating into the design, if you want to save the entire design, "
+"you need to back                       to the top level. If you want to "
+"export this module as new file, unlock the module and \"save as\""
 msgstr ""
-"You are editing a submodule, if you save it, you save only the submodule (in "
-"this situation \"save as\" works like \"export module\"), Do you like to "
-"continue?"
+"You are navigating into the design, if you want to save the entire design, "
+"you need to back                       to the top level. If you want to "
+"export this module as new file, unlock the module and \"save as\""
```

Comment:
"You are navigating into the design: If you want to save the entire design, "
"you need to go back                       to top-level. If you want to "
"export this module as a new file, unlock the module and \"Save as\"."

(*Guys: Should this be "You have navigated into..." or "You are navigating into..."? - not sure)

Reply:
or "... use \"Save as\" "

Reply:
I used this:
'You are navigating into the design: if you want to save the entire design, you need to go back to the top-level. If you want to export this module as new file, unlock the module and use "Save as"'

About the part of "You have navigated...", I can't tell: I'm not familiar with this part/feature of the IDE (yet).

Let me know if any should be changed.